sdk/translation/azure-ai-translation-text/azure/ai/translation/text/_operations/_operations.py

```diff
@@ -0,0 +1,1420 @@
+
+    return HttpRequest(method="POST", url=_url, params=_params, headers=_headers, **kwargs)
+
+
+def build_text_translation_lookup_dictionary_entries_request(
+    *, from_parameter: str, to: str, client_trace_id: Optional[str] = None, **kwargs: Any
+) -> HttpRequest:
+    _headers = case_insensitive_dict(kwargs.pop("headers", {}) or {})
+    _params = case_insensitive_dict(kwargs.pop("params", {}) or {})
+
+    content_type: Optional[str] = kwargs.pop("content_type", _headers.pop("Content-Type", None))
+    api_version: Literal["3.0"] = kwargs.pop("api_version", _params.pop("api-version", "3.0"))
+    accept = _headers.pop("Accept", "application/json")
+
+    # Construct URL
+    _url = "/dictionary/lookup"
+
+    # Construct parameters
+    _params["from"] = _SERIALIZER.query("from_parameter", from_parameter, "str")
+    _params["to"] = _SERIALIZER.query("to", to, "str")
+    _params["api-version"] = _SERIALIZER.query("api_version", api_version, "str")
+
+    # Construct headers
+    if client_trace_id is not None:
+        _headers["X-ClientTraceId"] = _SERIALIZER.header("client_trace_id", client_trace_id, "str")
+    _headers["Accept"] = _SERIALIZER.header("accept", accept, "str")
+    if content_type is not None:
+        _headers["Content-Type"] = _SERIALIZER.header("content_type", content_type, "str")
+
+    return HttpRequest(method="POST", url=_url, params=_params, headers=_headers, **kwargs)
+
+
+def build_text_translation_lookup_dictionary_examples_request(
+    *, from_parameter: str, to: str, client_trace_id: Optional[str] = None, **kwargs: Any
+) -> HttpRequest:
+    _headers = case_insensitive_dict(kwargs.pop("headers", {}) or {})
+    _params = case_insensitive_dict(kwargs.pop("params", {}) or {})
+
+    content_type: Optional[str] = kwargs.pop("content_type", _headers.pop("Content-Type", None))
+    api_version: Literal["3.0"] = kwargs.pop("api_version", _params.pop("api-version", "3.0"))
+    accept = _headers.pop("Accept", "application/json")
+
+    # Construct URL
+    _url = "/dictionary/examples"
+
+    # Construct parameters
+    _params["from"] = _SERIALIZER.query("from_parameter", from_parameter, "str")
+    _params["to"] = _SERIALIZER.query("to", to, "str")
+    _params["api-version"] = _SERIALIZER.query("api_version", api_version, "str")
+
+    # Construct headers
+    if client_trace_id is not None:
+        _headers["X-ClientTraceId"] = _SERIALIZER.header("client_trace_id", client_trace_id, "str")
+    _headers["Accept"] = _SERIALIZER.header("accept", accept, "str")
+    if content_type is not None:
+        _headers["Content-Type"] = _SERIALIZER.header("content_type", content_type, "str")
+
+    return HttpRequest(method="POST", url=_url, params=_params, headers=_headers, **kwargs)
+
+
+class TextTranslationClientOperationsMixin(TextTranslationClientMixinABC):
+    @distributed_trace
+    def get_languages(
+        self,
+        *,
+        client_trace_id: Optional[str] = None,
```

Comment:
It was discussed during APIView review for all languages. It was agreed that it is fine for first beta and for GA, it will be replaced with the custom HttpPolicy that would handle this special MT header.